catalog/core/catalog-core-standardframework/src/test/groovy/ddf/catalog/impl/operations/SourceOperationsTest.groovy

```diff
@@ -46,6 +47,26 @@ class SourceOperationsTest extends Specification {
         fedSources = ['fed1', 'fed2'].collectEntries { [(it): mockFedSource(it)] }
         pollerRunner = new SourcePollerRunner()
         sourcePoller = new SourcePoller(pollerRunner)
+        for (FederatedSource federatedSource : fedSources.values()) {
+            pollerRunner.bind(federatedSource)
+        }
+        pollerRunner.bind(catalogProviders.get(0))
+        int wait = 0
+        while (wait < 5) {
+            for (FederatedSource source : fedSources.values()) {
+                CachedSource cachedSource = sourcePoller.getCachedSource(source)
+                if (cachedSource == null || !cachedSource.isAvailable()) {
+                    Thread.sleep(100)
```

Comment:
:-(

If we don't already have a ticket to get rid of those sleeps, can we create one?